### PR TITLE
Fix guide-ch_bus_features.xml for translation

### PR DESCRIPTION
**Pull request by usamimikamen-sk**

### CODE REVIEW

**Review by fellen — COMMENTED** [review]
A good start!

Watch also overlapping with [Help:Business](https://github.com/Gnucash/gnucash-docs/blob/maint/help/C/Help_ch_Business.xml)
The technical parts like full description of fields in dialogs and their relations should go there.
OTOH the guide should collect the <q>cooking recipes</q>.

Watch also  https://wiki.gnucash.org/wiki/Business_Features_Issues.

- Are there updates mentioned?
- Can we move parts from the wiki into the docs?

Perhaps the wiki pages should get in their intro
`Before you continue here, you should already have read https://code.gnucash.org/docs/C/<component>/<chapter>.`
[Edit] and in the docs a link like
`Users might have collected updates and additional infos in <ulink url="&url-wiki-page;"/>`
Feel free to add missing pages into the DTD.

File: guide/C/ch_bus_features.xml [hunk 1]
```diff
@@ -158,20 +156,18 @@
...
-        Options</emphasis> window which can be accessed by
+      <para>The default tax table entry for new customers or new vendors can be specified in the <guilabel>Book Options</guilabel>
+        window which can be accessed by
         <menuchoice>
-          <guimenu>File</guimenu> <guisubmenu>Properties</guisubmenu> <guimenuitem>Business
-          tab.</guimenuitem>
-        </menuchoice>
+          <guimenu>File</guimenu><guimenuitem>Properties</guimenuitem><guimenuitem>Business</guimenuitem></menuchoice> tab.
+        
       </para>
```
[the remark this answers is not in the document]
Comment (usamimikamen-sk) [reply guide/C/ch_bus_features.xml:164]: No.
I can't afford to add accellerations and shortcuts into menuchoices. And I don't think they are not necessary to
be written in Concepts Guide. Write them in the Help Manual instead.

If you wan't to write them both of Guide and Help, create entity import system like
[https://github.com/Gnucash/gnucash-docs/tree/maint/xsl/1.79.2/common](https://github.com/Gnucash/gnucash-docs/tree/maint/xsl/1.79.2/common).

File: guide/C/ch_bus_features.xml [hunk 2]
```diff
@@ -237,9 +233,7 @@
           <para>If you set up Tax on Purchases and Tax on Sales as subaccounts of Liabilities:Tax then the net tax
             will be rolled up and can be seen in the &app; Accounts tab.
           </para>
-
-          <para><emphasis>If unsure about tax law requirements, get professional advice. </emphasis>
-          </para>
+        <caution><simpara>If you are unsure about tax law requirements, get the professional advices.</simpara></caution>
```
[the remark this answers is not in the document]
Comment (usamimikamen-sk) [reply guide/C/ch_bus_features.xml:240]: The docbook guide says, "(Caution might imply the possibility of harm to equipment whereas Warning might imply harm to a person), but DocBook makes no such assertions."
So I will replace caution tag with warning tag.

File: guide/C/ch_bus_features.xml [hunk 3]
```diff
@@ -1051,7 +1044,7 @@
         </figure>
 
         <note>
-          <para>You can modify the appearance of the invoice, IE: add a company logo, etc. To do so, see the
+          <para>You can modify the appearance of the invoice, That is add a company logo, etc. To do so, see the
```
[the remark this answers is not in the document]
Comment (usamimikamen-sk) [reply guide/C/ch_bus_features.xml]: I replaced the comma with a period.

File: guide/C/ch_bus_features.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jralls) [reply guide/C/ch_bus_features.xml]: Advices is not an English word. Advice is correct.

It looks like the only presentation difference between `caution` and `warning` is the icon associated with each.  Is it really appropriate in this case to break out that disclaimer as special for this particular section? The whole Guide should have a disclaimer at the top and bottom of every page: "The authors of this Guide have no formal accounting experience or credentials and cannot be relied upon. All users should consider obtaining the advice of a locally licensed professional accountant."

File: guide/C/ch_bus_features.xml [hunk 4]
```diff
@@ -555,30 +548,29 @@
...
+          </simpara>
+        </step>
 
-        <listitem>
-          <para>Enter your company&rsquo;s tax payer id number in the <guilabel>Company ID</guilabel> field.
-          </para>
-        </listitem>
+        <step><simpara>Enter your company&rsquo;s tax payer id number in the <guilabel>Company ID</guilabel> field.</simpara></step>
+      
+        <step><simpara>Select <guilabel>Default Customer Tax Table</guilabel> and <guilabel>Default Vendor Tax Table</guilabel>
```
Comment (usamimikamen-sk) [inline guide/C/ch_bus_features.xml:562]: In the GnuCash msgid, the word "TaxTable" is used. But it should be replaced with "Tax Table" for unification.

File: guide/C/ch_bus_features.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (usamimikamen-sk) [reply guide/C/ch_bus_features.xml]: advice is uncountable

[the remark this answers is not in the document]
Comment (usamimikamen-sk) [reply guide/C/ch_bus_features.xml]: 3fa3e5b

File: guide/C/ch_bus_features.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (fellen) [reply guide/C/ch_bus_features.xml:163]: Thanks, https://bugs.gnucash.org/show_bug.cgi?id=798273
For now we should add them to the DTD as
```
<ENTITY mnu-f-t "<menuchoice>
  <shortcut><keycombo>
    <keycap>Alt</keycap><keycap><keycap function="enter">Enter</keycap></keycap>
  </keycombo></shortcut>
  <guimenu><accel>F</accel>ile</guimenu><guimenuitem>Proper<accel>t</accel>ies</guimenuitem>
/menuchoice>">
```

**Pull request closed without merging**